bin/millau/node/src/service.rs

```diff
@@ -107,15 +109,15 @@ pub fn new_partial(
 	);
 
 	let (client, backend, keystore_container, task_manager) =
-		sc_service::new_full_parts::<Block, RuntimeApi, Executor>(
-			config,
+		sc_service::new_full_parts::<Block, RuntimeApi, _>(
+			&config,
 			telemetry.as_ref().map(|(_, telemetry)| telemetry.handle()),
 			executor,
 		)?;
 	let client = Arc::new(client);
 
 	let telemetry = telemetry.map(|(worker, telemetry)| {
-		task_manager.spawn_handle().spawn("telemetry", worker.run());
+		task_manager.spawn_handle().spawn("telemetry", None, worker.run());
```

Comment:
`SpawnTaskHandle` introduced groups. I've set all to default(None).

Reply:
Millau node is (mostly) a copy of https://github.com/paritytech/substrate/tree/master/bin/node-template . So if it is`None` there, then it should be fine